### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by lorenzoworx**
## Learning objectives
- Understand  how to create UIs adaptable to different screen sizes using media queries.
- Use media queries to make sure that your website displays correctly on desktop screens.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -142,16 +144,15 @@ footer > div > ul > li { margin-right: 21px; }
...
+.grid2 > div > h1 { padding-top: 12px; }
+.grid2 > div > p { padding-bottom: 12px; }
 
 .skills > li > ul > li {
   text-align: center;
   list-style: none;
   padding-bottom: 32px;
 }
-.grid2 > .techList > li { margin-right: 12px; }
+.grid2 > div > .techList > li { margin-right: 12px; }
```
Comment (BregornOriginal) [inline styles.css:162]: - [ ] Kindly fix this over widht on the tech list of your works section to fit inside the card as the FIGMA design

image

File: styles.css [hunk 2]
```diff
@@ -343,3 +346,276 @@ footer {
...
+  nav > img { display: none; }
+
+  section#works div.worksGrid div.grid1 {
+    position: relative;
+  }
+
+  section#works div.worksGrid div.grid1 hr {
+    display: none;
+  }
+
```
Comment (BregornOriginal) [inline styles.css:605]: - [ ] Kindly look at the FIGMA design line in the about me section, it doesn't go until the max width. Please reduce the size of it to fit on the FIGMA design 💯 

image

File: styles.css [hunk 3]
```diff
@@ -343,3 +346,276 @@ footer {
...
+  #about {
+    background-image: url("img/deskAbout.png");
+    background-repeat: no-repeat;
+    background-position: 100% 8.3%;
+  }
+
+  .aboutMe { width: 39.3vw; }
+
+  .skills {
```
Comment (BregornOriginal) [inline styles.css:537]: Good work implementing the skills section, but you have to do some changes to fit the FIGMA design

- [x] Kindly give each section more space between each list of items
- [x] Kindly align the list of items to left on each section

image

File: styles.css [hunk 4]
```diff
@@ -343,3 +346,276 @@ footer {
...
+  }
+
+  .deskNav > ul > li > a {
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    color: #fff;
+    font-weight: 600;
+    line-height: 20px;
+  }
+
```
Comment (BregornOriginal) [inline styles.css:587]: - [x] Kindly add more space between the social icons on the footer to fit the FIGMA design

image

File: styles.css [hunk 5]
```diff
@@ -343,3 +346,276 @@ footer {
...
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+  }
+
+  #works {
+    display: flex;
+    padding: 135px 9.9vw 130px 9.7vw;
+    height: auto;
+  }
```
Comment (BregornOriginal) [inline styles.css:413]: - [ ] On the works section, remember to follow the margin and padding on each element (you can check them on the template on FIGMA) so it has the exact space between each element, in this way the design will look better and have more attraction on the view of the visitor


image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @lorenzoworx  

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!

cat-driving-serious

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Desktop version looks very similar to FIGMA design
✅ Good use of GitHub Flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (lorenzoworx) [reply styles.css:162]: Can you please specify at what width this is happening because it looks good on my end

Comment (lorenzoworx) [reply styles.css:162]: Screenshot from 2022-12-10 00-23-40

Comment (BregornOriginal) [reply styles.css:162]: Hi Lorenzo, you can check it using the panel of developers on your browser.

image

**Review by pinkmoon25 — APPROVED** [review]
Hi @lorenzoworx , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 Linters are passing :heavy_check_mark: 
 Good commit history :heavy_check_mark: 
 Desktop version looks very similar to FIGMA design :heavy_check_mark: 
 Good use of GitHub Flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**